src/download_labels.py

```diff
@@ -1,143 +1,143 @@
     def __init__(self, extract_type='highway'):
-      '''
-          extract_type can so far be in: highway, tennis
-      '''
-      o.SimpleHandler.__init__(self)
-      self.ways = []
-      self.way_dict = {}
-      self.types = []
-      self.extract_type = extract_type
+        '''
+            extract_type can so far be in: highway, tennis
+        '''
+        o.SimpleHandler.__init__(self)
+        self.ways = []
+        self.way_dict = {}
+        self.types = []
+        self.extract_type = extract_type
 
     def way(self, w):
-      if self.extract_type == 'tennis':
-        self.extract_if_tennis_court(w)
-      else:
-        self.extract_way_type(w)
+        if self.extract_type == 'tennis':
+            self.extract_if_tennis_court(w)
+        else:
+            self.extract_way_type(w)
 
     def extract_if_tennis_court(self, w):
-      name = ''
-      is_tennis = False
-      for tag in w.tags:
-        if tag.k == 'sport' and 'tennis' == tag.v:
-          is_tennis = True
-        if tag.k == 'name':
-          name = tag.v
-
-      if not is_tennis:
-        return
-
-      way_dict = {
-            'uid': w.uid,
-            'ends_have_same_id': w.ends_have_same_id(),
-            'id': w.id,
-            'tags':[]}
-      for tag in w.tags:
-        way_dict['tags'].append((tag.k, tag.v))
-
-      self.add_linestring(w, way_dict)
+        is_tennis = False
+        for tag in w.tags:
+            if tag.k == 'sport' and 'tennis' == tag.v:
+                is_tennis = True
+
+        if not is_tennis:
+            return
+
+        way_dict = {'uid': w.uid,
+                    'ends_have_same_id': w.ends_have_same_id(),
+                    'id': w.id,
+                    'tags': []}
+
+        for tag in w.tags:
+            way_dict['tags'].append((tag.k, tag.v))
+
+        self.add_linestring(w, way_dict)
 
     def extract_way_type(self, w):
-      should_extract = False
-      name = ''
-      way_type = None
-      for tag in w.tags:
-        if tag.k == 'name':
-          name = tag.v
-        if tag.k == self.extract_type:
-          way_type = tag.v
-          should_extract = True
-
-      if not should_extract:
-        return
-      
-      if not way_type in self.types:
-        self.types.append(way_type)
-
-      way_dict = {'visible': w.visible,
-                  'deleted': w.deleted,
-                  'uid': w.uid,
-                  'way_type': way_type,
-                  'ends_have_same_id': w.ends_have_same_id(),
-                  'id': w.id,
-                  'tags':[]}
-      for tag in w.tags:
-        way_dict['tags'].append((tag.k, tag.v))
-      self.add_linestring(w, way_dict)
+        should_extract = False
+        way_type = None
+        for tag in w.tags:
+            if tag.k == self.extract_type:
+                way_type = tag.v
+                should_extract = True
+
+        if not should_extract:
+            return
+
+        if way_type not in self.types:
```

Comment:
Yeah, you are right, that was a bug. We weren't using the types, so it wasn't showing up.